<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagramConfiguration</class>
 <widget class="QMainWindow" name="DiagramConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>468</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">
     QSplitter::handle::horizontal, QSplitter::handle::vertical { width: 6px; background-color: #666; }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget_">
   <layout class="QVBoxLayout" name="vboxlayout_">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="wrapper">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>false</bool>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QTreeWidget" name="tree_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="SvgView" name="svg_view_" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu_">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="exitAction_"/>
   </widget>
   <widget class="QMenu" name="actionsMenu_">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="revealChildrenAction_"/>
    <addaction name="separator"/>
    <addaction name="exportAction_"/>
    <addaction name="separator"/>
    <addaction name="togglePreviewAction_"/>
    <addaction name="toggleLayoutAction_"/>
   </widget>
   <addaction name="fileMenu_"/>
   <addaction name="actionsMenu_"/>
  </widget>
  <action name="exitAction_">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="revealChildrenAction_">
   <property name="text">
    <string>Reveal Children</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="exportAction_">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="togglePreviewAction_">
   <property name="text">
    <string>Toggle UML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="toggleLayoutAction_">
   <property name="text">
    <string>Toggle Preview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SvgView</class>
   <header>INCode.widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
